validation/open_boundaries/mode_one_internal_wave.jl

```diff
@@ -0,0 +1,111 @@
+using Oceananigans
+using Oceananigans.Fields: FunctionField
+
+# TODO: clean up all the global variables in this script
+
+h = 10
+k = π/h
+
+λ = 2π / k
+
+grid = RectilinearGrid(topology = (Bounded, Flat, Bounded), size = (256, 128), extent = (3*λ, h))
+
+xc, yc, zf = nodes(grid, Center(), Center(), Face())
+xf, yc, zc = nodes(grid, Face(), Center(), Center())
+
+N² = 0.2
+w₀ = .03
+
+ω = @show sqrt(k^2 * N² / (k^2 + (π/h)^2))
+
+u(x, z, t) = w₀ * π / (k * h) * cos(π * z / h) * cos(k * x - ω * t)
+
+w(x, z, t) = w₀ * sin(π * z / h) * sin(k * x - ω * t)
+
+b(x, z, t) = - N² * w₀ / ω * sin(π * z / h) * cos(k * x - ω * t) + N² * z
+
+@inline u_east(z, t) = u(xf[grid.Nx], z, t)
```

Comment:
Minor thing, but shouldn't the BC here be the forcing at `xf[Nx+1]` such as below?

```suggestion
@inline u_east(z, t) = u(xf[grid.Nx+1], z, t)
```